### ISSUE TITLE
16 - A drawer needs to draw all words correctly to get full points

### ISSUE THREAD

**Issue by SyLi9527** [sopra-fs22-group-27/Pictionary-Server#46]
In a game, more than one word may be shown to the user. In this case, the user needs to draw these words correctly to get full points.

## Acceptance Criteria:
- [ ] If the user draws all words, he/she will get full points.
- [ ] Otherwise, if the user draws at least one word correctly, he/she will get half points.

**Comment by SyLi9527** [comment]
@radubauzh Is it possible to show two words in one round for 'drawers' to draw? My idea was that if a drawer could draw two words that both receive a good score calculated by google vision API(for example, both>0.8), the drawer would receive more points than normal since the difficulty is higher than drawing just one word and the correct guesser is less than normal. Is it reasonable?

**Comment by radubauzh** [comment]
@SyLi9527  Yes this sounds reasonable. Not sure if we will find time for it but you already marked it as optional so I think this should be good :)

**Comment by SyLi9527** [comment]
@radubauzh Cool. I'll do it if time permits :)
